src/Controllers/GagePageController.ts

```diff
@@ -248,8 +248,20 @@ module StreamStats.Controllers {
         public formattedDailyPlusAvg = [];
         public formattedDischargePeakDates = []; // Stage vs. Discharge Plot
         public dailyValuesOnly = [];
-        public ageQualityData = 'age'; //Stage vs. Discharge Plot
-        public error: any; //Stage vs. Discharge Plot
+        public ageQualityData = 'age';
+        public error: any;     
```

Comment:
I think a more descriptive label for this variable might be helpful.

Reply:
Agreed! However, changing error to a different name and it's associated references, was screwing things up. Not sure if I should spend more time on this and figure it out?

Reply:
mm not sure why that would happen. Did you try changing:

line 252 to `public measuredError: any;`
line 1090 to `this.measuredError = data.includes(errorMessage)`
line 1091 to `if (this.measuredError == false) {`
line 3189 to `showInLegend: this.measuredError == false`

Reply:
Yea those exact lines after finding all references. I change it to any other variable name and then I lose my age and quality buttons.